### DISCUSSION TITLE
How to automate the execution of vscode activitybar window move to the Secondary Sidebar.

### DISCUSSION THREAD

**Discussion by Anthonywck** [microsoft/vscode-discussions#2462] [Extension Development QnA] [answered]
Hi, there:
I am currently developing a Visual Studio Code extension and am seeking a method to automatically transfer the webview page, which is initially loaded on the Activity Bar, to be displayed in the Secondary Sidebar, as illustrated below.

![移动视图](https://github.com/user-attachments/assets/e7615e0f-7d17-43ff-aa05-e37b9de5df07)

I identified a comparable Visual Studio Code command, however, it did not function as effectively as anticipated.
```typescript
await vscode.commands.executeCommand('workbench.action.moveView', {
      viewIds: ['lightCode'], // 源视图 ID
      destinationId: 'workbench.view.sidebar' // 目标容器 ID
});
```
The current results of this approach are as follows:
![移动视图-command](https://github.com/user-attachments/assets/d44a9ecd-38fd-4aa9-b6b2-ef6f200db568)
How can this effect be automatically achieved after the plugin is loaded?

Thanks for your attention and reply.

**Comment by alefragnani** [comment, marked as answer]
Hi @Anthonywck ,

Do you _actually_ need move to the Secondary Side Bar, or are you doing this move because you can't define the Secondary Side Bar as the default location for your WebView?

If that's the case, I suggest you to open a feature request in the VS Code repo, so the VS Code team could evaluate and expose a new API/contribution point, if approved.

Hope this helps

**Reply by Anthonywck** [reply]
@alefragnani That's why I brought this up, and thank you for your advice.
